README.md

```diff
@@ -6,33 +6,52 @@
 This set of manifests gets a local obersvability stack up and running quickly.
 It installs the following services into your local kubernetes cluster:
 
-* Grafana
-* Prometheus
-* Tempo
-* Loki
-* Certificate Manager
-* OpenTelemetry Controller
-* Liatrio OpenTelemetry Collector
-* NGrok Ingress and API Gateway Controller
+- Grafana
+- Prometheus
+- Tempo
+- Loki
+- Certificate Manager
+- OpenTelemetry Controller
+- Liatrio OpenTelemetry Collector
+- NGrok Ingress and API Gateway Controller
 
 ## Prerequisites
 
 > OSX users with [Homebrew][brew] installed can install the Prerequisites by running the command `brew bundle`
 
 1. Run kubernetes locally. Here are a few options:
    1. [Docker Desktop][dd]: Local instance of Docker and k8s.
-   2. [k3d][k3d]: a lightweight wrapper to run k3s (Rancher Lab’s minimal k8s distribution) in docker.
+   2. [k3d][k3d]: a lightweight wrapper to run k3s (Rancher Lab’s minimal k8s distribution) in docker. (Required if using tilt)
 2. Have kubectl installed
 3. Have kustomize installed
-4. If using DORA, have NGROK configured with a domain and update the configuration accordingly.
-5. Have a free NGrok Account with a Permanent domain (if wanting to deploy DORA)
-6. Have helm installed (gross, only for the ngrok helm chart, will remove this eventually)
+4. Have [tilt][tilt] installed (Optional)
```

Comment:
Let's include a `Brewfile` that has `tilt` defined in it so it's easy to install with `brew bundle` and make sure it's called out here. I think that's already done for k3d.

Reply:
There's an existing Brewfile in the repo. I added Tilt to it.